src/main.js

```diff
@@ -47,15 +47,16 @@ fs.readdir("./src/commands", function(err, files)
 		if(data.nick == config.nick)
 			return;
 
-		if(this.bans.indexOf(data.nick.toLowerCase()) !== -1)
-		{
-			this.send(data.nick + " you are banned.");
-			return;
-		}
+		
 
 		var msg = data.text;
 		if(msg[0] == "!")
 		{
+			if(this.bans.indexOf(data.nick.toLowerCase()) !== -1)
+			{
+				this.send(data.nick + " you are banned.");
```

Comment:
banning is for preventing them to spam so telling them that they are banned every time just makes no sense...